kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/AccordionContainer.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+
+  <div>
+    <div>
+      <DragContainer
+        :items="items"
+      >
+        <transition-group
+          tag="div"
+          name="list"
+          class="wrapper"
+        >
+          <slot
+            :toggleItemState="toggleItemState"
+            :isItemExpanded="isItemExpanded"
+            :isOptionSelected="isOptionSelected"
+            :isAnswerSelected="isAnswerSelected"
+          ></slot>
+        </transition-group>
+      </DragContainer>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  // import DragSortWidget from 'kolibri.coreVue.components.DragSortWidget';
+  import DragContainer from 'kolibri.coreVue.components.DragContainer';
+
+  export default {
+    name: 'AccordionContainer',
+    components: {
+      DragContainer,
+    },
+    props: {
+      items: {
+        type: Array,
+        required: true,
+      },
+    },
+    data() {
+      return {
+        itemIds: [],
```

Comment:
Maybe we ought to call this `expandedItemIds` to make clear what we're tracking here